attributes/customized-multipath.rb

```diff
@@ -0,0 +1,4 @@
+if node['multipath']['storage_type'] == "customized"
```

Comment:
I think it would be better to put this inside of a hash rather than using the if matching here.

ex:

```
default['multipath']['customized']['defaults']
```